client/src/app/app.component.scss

```diff
@@ -0,0 +1,37 @@
+// Standard Styles
+body {
+    /* Margin bottom by footer height */
+    margin-bottom: 60px;
+  }
+  .footer {
```

Comment:
You've defined the `footer` class, but you haven't defined the `header` class as mentioned in the design document.